.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [19.x, 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
+        node: [20.x, 18.x, 16.x, 14.x, 12.x, "12.22.0"]
```

Comment:
Can we keep 17 and 19, like in https://github.com/eslint/eslint/pull/17093?

(the same for other packages)

Reply:
Yes, updating my PRs accordingly.

Reply:
```suggestion
        node: [20.x, 19.x 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
```